data.json

```diff
@@ -964,5 +964,11 @@
     "githubUsername": "gagan3012",
     "imageUrl": "https://user-images.githubusercontent.com/49101362/120364787-7c501880-c2c2-11eb-8f18-0f0ef10c4439.png",
     "issueNumber": 539
+  },
+  {
+    "name": "Aditya Mangal",
+    "githubUsername": "adityamangal1",
+    "imageUrl": "https://user-images.githubusercontent.com/68494604/120438793-6bc1a000-c39f-11eb-845b-c145db5e2f9b.png",
+    "issueNumber": 544
```

Comment:
LGTM

Reply:
I don't know what's the problem then now I created again a pull request. @skjha1